src/components/TransactionItemRow/index.tsx

```diff
@@ -46,24 +52,29 @@ function TransactionItemRow({
 
     const {selectionMode} = useMobileSelectionMode();
 
+    const merchantName = getMerchant(transactionItem);
+    const isMerchantEmpty = merchantName === CONST.TRANSACTION.PARTIAL_TRANSACTION_MERCHANT;
+
     return (
         <View style={styles.flex1}>
             {shouldUseNarrowLayout ? (
                 <Hoverable>
                     {(hovered) => (
-                        <View style={[hovered ? styles.hoveredComponentBG : backgroundColor, styles.expenseWidgetRadius, styles.justifyContentEvenly, styles.gap3]}>
+                        <View style={containerStyles ?? [hovered ? styles.hoveredComponentBG : backgroundColor, styles.p2, styles.expenseWidgetRadius]}>
+                            {' '}
```

Comment:
Why do we need this?

Reply:
These are remnants of the now-removed and no longer needed circle used for selecting a row on its right side.